corehq/apps/domain/decorators.py

```diff
@@ -319,7 +331,8 @@ def _inner(request, domain, *args, **kwargs):
             domain_name, domain = load_domain(request, domain)
             if not domain:
                 raise Http404()
-            if not request.couch_user.is_domain_admin(domain_name):
+            if not _page_is_whitelist(request.path, domain_name) and \
```

Comment:
hmm, i see I was confused about what this was doing (this is the "no permission" check). why was this necessary? it seems like this is just more restrictive than before (i made this truth table because i was having a hard time wrapping my head around this).

![image](https://cloud.githubusercontent.com/assets/66555/15651000/a6cb051c-264a-11e6-9ef5-bcb5ce873ff9.png)